third_party/py/python_configure.bzl

```diff
@@ -138,10 +138,13 @@ def _symlink_genrule_for_dir(repository_ctx,
 
 def _get_python_bin(repository_ctx):
     """Gets the python bin path."""
-    python_bin = repository_ctx.os.environ.get(_PYTHON_BIN_PATH)
-    if python_bin != None:
-        return python_bin
-    python_bin_path = repository_ctx.which("python")
+    python_bin = repository_ctx.os.environ.get(_PYTHON_BIN_PATH, 'python')
+    if not '/' in python_bin and not '\\' in python_bin:
```

Comment:
Maybe `not os.path.isabs(python_bin)`?

Reply:
I wasn't sure if user will only pass absolute path to `PYTHON_BIN_PATH`. In our `tools` shell files, the environment variable `PYTHON` often set to relative path to point to the virtual environment Python binary.

Reply:
Understood. My point is that we should avoid writing special cases for different platforms wherever possible.

Reply:
I don't like to check it in this way either... Do you have better solution than the two slash checks?

Reply:
Updated to `if not repository_ctx.path(python_bin).exists:`, seems this is the way Skylark (Starlark) check file existence.